src/mono/wasm/runtime-test.js

```diff
@@ -248,7 +248,9 @@ function set_exit_code(exit_code, reason) {
     }
     if (is_browser) {
         // tell xharness stream procesor we are done
-        originalConsole.log("WASM EXIT " + exit_code);
+        setTimeout(() => {
```

Comment:
Does this need to be 200 specifically? Or are you still troubleshooting

Reply:
I'm still troubleshooting.
I still don't know how the C# code could stop in the middle of reporting TestResults.xml
It was missing even [`After RunTests`](https://github.com/dotnet/runtime/pull/61313/files#diff-86185eb9de15ff10811d033f377f65f9ca6ee32bc5d90d95b7f95b66fd2aae46R75) log line on previous build